docs/SMOKE_TESTING.md

```diff
@@ -0,0 +1,26 @@
+
+# Running Smoke Tests for Forest's RPC API
+
+## Prerequisites
+In order to smoke test the RPC API endpoints, you need to meet 2 pre-requisites.
+* A forest node should be running locally on port `1234`
+* `FULLNODE_API_INFO` needs to be set as an environment variable. This can be done
+   as a prefix to the actual command, set with a script that is sourced (linux),
+   or set via the command line. If you don't know the multiaddr of the node, you can
+   use `forest auth api-info` to get the key-value pair needed.
```

Comment:
I like it, let me see if I can wrangle something like that together. Currently permissions are not being tested. These smoke tests are just testing if the endpoints can be hit and a response is returned.

Reply:
Cool! And yeah, it's fine to not test permissions for a first stab at this.

Reply:
Smoke tests should not test permissions. If we want to test permissions we should set up some auth testing which would be similar to integration tests

Reply:
Honestly, I'm not sure there's a decent way to do a comprehensive evaluation beyond a basic validation of the auth mechanism itself since the Lotus OpenRPC schema doesn't specify permissions.

I do think auth levels should be verified against at least one endpoint through smoke tests, but I'm fine with you implementing that later or some other way.

Reply:
Implemented the requested changes. Should be ready now

Reply:
Cool, I'll give it another look

Reply:
@connormullett Also, can you update the docs for this? I don't think we need the env variable now, right?

Reply:
You are right, we don't need that anymore. Good catch!